mongo_connection.ts

```diff
@@ -0,0 +1,29 @@
+import { MongoClient, Db } from 'mongodb';
+
+const readEnv = (env: string) => {
```

Comment:
Wouldn't there be an error thrown by the parser if there is a missing value? Did you test this.


A good option would be to add default values to the config so you won't have these errors